astropy/utils/iers/iers.py

```diff
@@ -786,7 +790,7 @@ def open(cls):
 
         """
         if not conf.auto_download:
```

Comment:
I guess the alternative path, which would preserve anything downloaded, would simply be to remove this stanza and in the `else` clause of the `for` loop warn only if `conf.auto_download` is set.

Am I missing something?

Reply:
Discussion concluded we should leave this for follow-up